packages/server-utils/src/sendEmail.ts

```diff
@@ -26,7 +26,10 @@ export const sendEmail = async (to: string | string[], mailOptions: MailOptions
     attachments,
     signOff = html ? HTML_SIGN_OFF : TEXT_SIGN_OFF,
   } = mailOptions;
-  const { SMTP_HOST, SMTP_USER, SMTP_PASSWORD, SITE_EMAIL_ADDRESS } = process.env;
+  const SMTP_HOST = requireEnv('SMTP_HOST');
+  const SMTP_USER = requireEnv('SMTP_USER');
+  const SMTP_PASSWORD = requireEnv('SMTP_PASSWORD');
+  const SITE_EMAIL_ADDRESS = requireEnv('SITE_EMAIL_ADDRESS');
```

Comment:
```suggestion
  const SMTP_HOST = getEnvVarOrDefault('SMTP_HOST', undefined);
  const SMTP_USER = getEnvVarOrDefault('SMTP_USER', undefined);
  const SMTP_PASSWORD = getEnvVarOrDefault('SMTP_PASSWORD', undefined);
  const SITE_EMAIL_ADDRESS = getEnvVarOrDefault('SITE_EMAIL_ADDRESS', undefined);
```
I think `undefined` is valid in this scenario. We just don't send emails if the SMTP config is missing